cosmos/pocket-testnet.json

```diff
@@ -0,0 +1,49 @@
+{
+
+  "chainId": "pocket-beta",
```

Comment:
The name of the file and the images/ directory should match chain id. https://github.com/chainapsis/keplr-chain-registry/actions/runs/14231617866/job/39883259588?pr=1081